overlord/snapstate/snapstate.go

```diff
@@ -536,6 +536,7 @@ func InstallMany(st *state.State, names []string, userID int) ([]string, []*stat
 			return nil, nil, err
 		}
 		installed = append(installed, name)
+		ts.JoinLane(st.NewLane())
```

Comment:
do we have a test for this?

Reply:
@stolowski can you please add a comment why this is relevant:
```
Handle each installation in a separate lane so that unrelated snaps can install successfully in face of failure to install a particular one.
```

CC: @mvo5 will this be a problem for your prerequisite work?

Reply:
@zyga It might have funny effects, afaict if the snap foo gets instaleld in a lane and it requires a prereq that prereq will get installed in the default lane. Which means that if foo fails the prereq is still instaleld. or worse if the prereq fails foo also gets installed. I think we need code in the prepreq install that puts the prereq into the same lane.

But stepping back, this change does not really fit into the title of the PR, maybe a separate PR so that its more clear what change is happening here.

Reply:
no, the prereqs are put in their own lane, not the default lane (that was intentional afair)

Reply:
@mvo Ok, I'll move this change (plus a tiny change to UpdateMany test) into a separate PR.